src/granularity/granularityBase.ts

```diff
@@ -80,6 +80,8 @@ export class TimelineGranularityBase implements IGranularity {
     private shortMonthFormatter: valueFormatter.IValueFormatter;
     private granularityProps: IGranularityName = null;
 
+    private DefaultQuarter: number = 3;
```

Comment:
why does it equal 3 by default?

Reply:
Looks like parsing of dates is related with reverse order of quarter index here. I just moved it from quarter granularity to base granularity without changing of the logic.